database/buildSchema/27_review_assignment.sql

```diff
@@ -1,19 +1,5 @@
 -- review assignment
--- FUNCTION to auto-add template_id to review_assignment
-CREATE OR REPLACE FUNCTION public.review_assignment_template_id (application_id int)
-    RETURNS int
-    AS $$
-    SELECT
-        template_id
-    FROM
-        application
-    WHERE
-        id = $1;
-
-$$
-LANGUAGE SQL
-IMMUTABLE;
-
```

Comment:
Matching the pattern used everywhere else to *only* define the function in the main "functions" script